CounterApp/Components/UserInfo/UserInfoCore.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  UserInfoCore.swift
+//  CounterApp
+//
+//  Created by Wenjuan Li on 2022/7/22.
+//
+
+import ComposableArchitecture
+
+struct UserInfoState: Equatable {
+    var user: User
+    var disabled: Bool = false
```

Comment:
I've been exploring with the idea of moving this type of variables to the view since this is merely view configuration that doesn't change at runtime.